app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -99,11 +99,15 @@ def self.local_prefixes
     private
 
     def update_referer
-      hyrax.edit_admin_admin_set_path(@admin_set) + (params[:referer_anchor] || '')
+      hyrax.edit_admin_admin_set_path(admin_set_id) + (params[:referer_anchor] || '')
     end
 
     def create_admin_set
-      admin_set_create_service.call(admin_set: @admin_set, creating_user: current_user)
+      updated_admin_set = admin_set_create_service.call!(admin_set: admin_set_resource, creating_user: current_user)
+      update_admin_set(updated_admin_set)
```

Comment:
This is required because @admin_set does not get updated by the create service when it is a `Hyrax::AdministrativeSet`.  The create service returns the updated admin set, so the instance variable update has to happen in the controller.